ui/modal/info_modal.go

```diff
@@ -171,22 +183,31 @@ func (in *InfoModal) HandleKeyEvent(evt *key.Event) {
 
 func (in *InfoModal) Handle() {
 	for in.btnPositve.Clicked() {
-		in.DismissModal(in)
+		if in.isLoading {
+			return
+		}
 		isChecked := false
 		if in.checkbox.CheckBox != nil {
 			isChecked = in.checkbox.CheckBox.Value
 		}
 
 		in.positiveButtonClicked(isChecked)
+		if !in.isLoading {
+			in.DismissModal(in)
+		}
```

Comment:
i dont think you need this here again. 
You already return when loading is set on line 186... 
```suggestion
	     in.DismissModal(in)

```

Reply:
That is needed else modal will always dismiss even when we need to show the loading widget. https://github.com/planetdecred/godcr/blob/84fa15f87dcd236993f5e402a4b56b2e48e28fb5/ui/modal/info_modal.go#L186 this is needed just in case someone is super fast and clicks the button twice before the loading widget is activated.

Reply:
what you have is a for loop meaning for every click, it starts a new instance.

Reply:
I didn't write the for loop. I only modified it's content. I'll read the connected codes to figure out if for loop is appropriate there.

Reply:
The review was not if you wrote it or if the for loop was wrong... it was for the redundant lines of codes to be removed.